api/controllers/PokemonController.js

```diff
@@ -25,6 +25,7 @@ module.exports = _.mapValues({
     let box;
     if (params.box) {
       box = await Box.findOne({id: params.box});
+      console.log(box);
```

Comment:
Oh yeah, it was for testing my box uploading was working (because we don't have any way to view the box yet).